docs/dev/tutorials/custom-paymaster-tutorial.md

```diff
@@ -265,11 +270,16 @@ contract MyPaymaster is IPaymaster {
         allowedToken = _erc20;
     }
 
-    function validateAndPayForPaymasterTransaction (
+    function validateAndPayForPaymasterTransaction(
         bytes32,
         bytes32,
         Transaction calldata _transaction
-    ) onlyBootloader external payable returns (bytes4 magic, bytes memory context) {
+    )
+        external
+        payable
+        onlyBootloader
```

Comment:
I believe (could be wrong) the modifier has to go at the beginning of this list.